nodeup/pkg/model/directories.go

```diff
@@ -29,11 +29,12 @@ type DirectoryBuilder struct {
 
 var _ fi.ModelBuilder = &DirectoryBuilder{}
 
+// Build is ensures an number of directories
```

Comment:
Bit confusing, did you mean something like: `Build ensures the Kubernetes bin directory is mounted and files executable on any changes`?